src/__tests__/JSL.test.ts

```diff
@@ -66,4 +66,20 @@ describe('JSL', () => {
       expect(returnedError[idx]).toEqual(testCase);
     });
   });
+
+  it('tests with empty jsl input', () => {
+    const input = '';
+    const json = { data: [{}, {}, { width: 200 }] };
+
+    try {
+      const program = JSL.compile(input);
+      const width = JSL.evaluate(json, program);
+
+      expect(program).toMatchSnapshot();
+      expect(width).toMatchSnapshot();
```

Comment:
Why do we need to deal with the overhead of snapshots?? these could be replaced with simply `expect(width).toBeNull();`